libevm/VM.h

```diff
@@ -97,19 +101,17 @@ class VM: public VMFace
 	// space for memory
 	bytes m_mem;
 
-	// space for code and pointer to data
-	bytes m_codeSpace;
-	byte* m_code = nullptr;
+	// space for code
+	bytes m_code;
 
-	// space for stack and pointer to data
-	u256 m_stackSpace[1025];
-	u256* m_stack = m_stackSpace + 1;
-	ptrdiff_t stackSize() { return m_SP - m_stack; }
+	// space for data stack, grows towards smaller addresses from the end
+	u256 m_stack[1024];
```

Comment:
`unsigned` type has to support value wrap (`int` not, because in signed types overflow is undefined behavior). So if compiler cannot prove that overflow with `unsigned` cannot happen it has to generate additional code to support the wrap, because `unsigned` is 32-bit and we are on 64-bit architecture :)